services/QuillLMS/config/initializers/secure_headers.rb

```diff
@@ -1,5 +1,55 @@
 SecureHeaders::Configuration.default do |config|
-  config.csp = SecureHeaders::OPT_OUT
+  config.csp = {
+    default_src: [
+      "'self'", 
```

Comment:
This double+single quote syntax is intentional. Consult one of the linked reference sites for more info.